packages/react/src/themes/teams-high-contrast/components/Attachment/attachmentStyles.ts

```diff
@@ -33,12 +33,6 @@ const attachmentStyles: ComponentSlotStylesInput<AttachmentProps, AttachmentVari
       },
     }),
   }),
-
-  action: ({ variables: v }): ICSSInJSStyle => ({
-    ':hover': {
```

Comment:
where these styles have migrated to? am I right that those are not needed now?

Reply:
indeed they are not needed and they were applied to the wrong element anyway ( wrapping div instead of stardust `Button` )